source/includes/_instruments.md

```diff
@@ -80,7 +80,17 @@ X-GOJI-REQUEST-ID: 79f33f3c-86e0-4613-ba49-9fac3c6f0eab
 
 ### Description
 
-Registers a new instrument which can then be referenced by it's symbol in other APIs which require an InstrumentSymbol.
+Registers a new instrument which can then be referenced by its symbol in other APIs which require an `InstrumentSymbol`.
+
+
+The instrument can then be referenced by the `InstrumentSymbol` the following APIs:
+
+ * [Trade Settlement API](/#trade-settlement)
+ * [Corporate Actions API](/#corporate-actions)
+ 
+In addition to defining the instrument, a list of fees that will apply to any trade settlement for this instrument 
+must be specified.  The `FeeSymbol`s that have been registered will later be referenced in the 
```

Comment:
wondering if '... can then be referenced in...' would be clearer to the reader that it is something they have to do instead of '...will later be referenced in...'

Reply:
I've updated this with a little more detail.  Can tidy up when we write the trade settlement docs. Happy with my changes? @victokoh